yaksh/views.py

```diff
@@ -31,6 +30,7 @@
                 QuestionFilterForm, CourseForm, ProfileForm, UploadFileForm,\
                 get_object_form, FileForm, QuestionPaperForm
 from .settings import URL_ROOT
+from django.conf import settings
```

Comment:
Why can't we just import the FIXTURE_DIRS from online_test.settings instead of django.conf settings?